test/metabase/models/data_permissions_test.clj

```diff
@@ -162,7 +177,20 @@
         (is (= :block (data-perms/table-permission-for-user user-id :perms/data-access database-id table-id-2))))
 
       (testing "Admins always have the most permissive value, regardless of group membership"
-        (is (= :unrestricted (data-perms/table-permission-for-user (mt/user->id :crowberto) :perms/data-access database-id table-id-2)))))))
+        (is (= :unrestricted (data-perms/table-permission-for-user (mt/user->id :crowberto) :perms/data-access database-id table-id-2)))))
+      (mt/with-restored-data-perms-for-groups! [group-id-1 group-id-2]
```

Comment:
```suggestion
        (is (= :unrestricted (data-perms/table-permission-for-user (mt/user->id :crowberto) :perms/data-access database-id table-id-2)))))
        
     (mt/with-restored-data-perms-for-groups! [group-id-1 group-id-2]
```

Reply:
(looks like this was at the same indentation as the previous `testing` block, but the parens weren't balanced, or something...not sure exactly but you might want to check the indentation here)